sphinx/templates/quickstart/conf.py_t

```diff
@@ -105,7 +105,7 @@ html_static_path = ['{{ dot }}static']
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+intersphinx_mapping = {'https://docs.python.org/3/': None}
```

Comment:
Why is this needed? If my understanding correct, `https://docs.python.org/` returns appreciate URL of python document. It is not only for 2.x.
```
$ curl -I https://docs.python.org/
HTTP/2 302
server: nginx
content-type: text/html
location: https://docs.python.org/3/
x-clacks-overhead: GNU Terry Pratchett
strict-transport-security: max-age=315360000; includeSubDomains; preload
via: 1.1 varnish
accept-ranges: bytes
date: Thu, 01 Aug 2019 15:47:56 GMT
via: 1.1 varnish
age: 112150
x-served-by: cache-jfk8140-JFK, cache-tyo19950-TYO
x-cache: HIT, HIT
x-cache-hits: 1, 3
x-timer: S1564674477.715171,VS0,VE0
content-length: 154
```

Reply:
It saves the network overhead of a second client request from the redirect: `HTTP/2 302`.

😆 : `x-clacks-overhead: GNU Terry Pratchett` oh, Python peeps are the best!

Reply:
Hmm... The title and purpose of this PR are not related with change, right? If true, I can understand what are you doing. Indeed, this reduces overhead of communication.